interface/main/tabs/css/menu.css

```diff
@@ -107,13 +107,43 @@ and open the template in the editor.
 /*
     Latest additions in order to make navbar theme independent by using Bootstrap.
 */
+
+
+
+.dropdown-menu {
+  
+    background-color: #004f87;
+    padding: 0px 0px;
+}
+.nav>li>a:focus, .nav>li>a:hover {
+    text-decoration: none;
+    background-color: #5cb5f3f2 !important;
+}
+.nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
+    background-color: #5cb5f3f2 !important;
+    border-color: #337ab7;
+}
 .nav>li>a{
-    color: #1057c9;
+    color: #ffff;
     cursor: pointer;
     font-size: 14px;
     font-weight : 600;
-    padding-right : 10px;
+    padding : 20px;
```

Comment:
Hi @Urvashimeena !
Thank you for the contribution!

 I see you increased the padding here a good bit.  This might be totally fine, (I am not yet testing it).  We do however have a vertical space worry, in that the more room taken up by a top navbar, the less data and more scrolling required.  Horizontal padding is fine, but excess vertical (unless reacting to a defined mobile device theme) is sort of the enemy.  We know it makes things nicer looking, but the application is kinda data-intensive.  You see that a lot more when testing with a large database, and working with multiple tabs open.

I recommend testing your changes with the data set additions located here:  https://github.com/LibreHealthIO/EHR_Contribs.